index.js

```diff
@@ -141,28 +194,53 @@ const line = []
 
 
 // *** Uncomment the lines below to test
-// console.log("%cScope & Closures - Question 1", "color: red")
+console.log("%cScope & Closures - Question 1", "color: red")
+
+let lastserverd = null
+
+function takeATicketNumber(line) {
+  if (line.length == 0 && lastserverd == null) {
+    line.push(1);
+    lastserverd = 1;
+  } 
+  else {
+    line.push(lastserverd + 1);
+    lastserverd++;
+  }
```

Comment:
Good! Don't forget to user camel case for your variable names, too! This can also be simplified to: 
```suggestion
let lastServerd = 0

function takeATicketNumber(line) {
       lastServerd += 1;
       line.push(1);
```